content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -31,12 +31,10 @@ The following API will no longer be supported:
 
 Instead, a time-to-live configuration or retention rules can be used to remove expired measurements data from the Operational Store.
 
-##### Breaking change in the Alarms, Events, Measurements APIs - unsupported query parameters will be rejected and required parameters will be introduced
+##### Breaking change in the Alarms, Events, Measurements APIs - required parameters will be introduced
 
-As of release 10.17+, it will no longer be possible to pass unsupported HTTP query parameters. This means that also all requests with wrongly typed parameters will be rejected.
-This change affects the Alarms, Events and Measurements APIs.
-
-At least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
+As of release 10.17+, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
```

Comment:
```suggestion
As of release 10.17, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
```